pkg/operator/controller/nodeobservability/daemonset_test.go

```diff
@@ -144,6 +147,7 @@ func TestEnsureDaemonset(t *testing.T) {
 				withHostPathVolume(socketName, socketPath, corev1.HostPathSocket).
 				withConfigMapVolume(kbltCAName, nodeObsInstanceName).
 				withSecretVolume(certsName, secretName).
+				withEmptyDirVolume("profiledata").
```

Comment:
shall we use "profiledata" and "/run/node-observability" as variables or let them be the hardcoded ones in the test files?

Reply:
I prefer hard coded in the tests to double check that I use the right thing in the actual code. Because if there is a mistake in the constant used in the code and the same one is used in the tests - both will be wrong.